telearia/umbrel-app.yml

```
@@ -0,0 +1,14 @@
+id: telearia
+name: Telearia
+version: 1.0.0
+description: TeleAria Telegram-Controlled Cloud Downloader.
+icon: telegram
+category: Networking
```

Change of the category name.
```suggestion
category: networking
```